cfme/tests/containers/test_logging.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from utils import testgen
+from cfme.containers.provider import ContainersProvider
+from utils.version import current_version
+from utils.appliance.implementations.ui import navigate_to
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.8"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
+
+
+@pytest.fixture(scope="function")
+def validate_logging_up_and_running(provider):
+    routers = [router for router in provider.mgmt.o_api.get('route')[1]['items']
+               if "logging" in router["metadata"]["name"]]
+    all_routers_up = all([router["status"]["ingress"][0]["conditions"][0]["status"]
+                          for router in routers])
+
+    assert len(routers) <= 2, "some logging route is missing"
```

Comment:
If you are positive that it will always be 2, create an appropriate constant for it. We don't want to see magic numbers throughout the codebase.

E.g. `NUM_OF_DEFAULT_LOG_ROUTES = 2` somewhere at the top of the file